src/main/java/pl/sda/weather/model/WeatherLine.java

```diff
@@ -0,0 +1,222 @@
+package pl.sda.weather.model;
+
+import java.util.Arrays;
+
+public class WeatherLine {
+    private City city;
+    private long time;
+    private MainWeather main;
+    private Wind wind;
+    private Clouds clouds;
```

Comment:
btw. you don't have to declare all fields (event those not needed) -> you can just ignore those during deserialization